src/components/SheetRenderer.js

```
@@ -149,21 +199,145 @@ export default class SheetRenderer {
+        payload = JSON.parse(overlayPayloadRaw);
+      } catch {
+        // Malformed payload – ignore.
+        return;
+      }
+
+      const { field, index } = payload || {};
+      if (!field || index == null) return;
+
+      const td = e.target.closest('td');
+      if (!td) return;
+      const row = parseInt(td.dataset.r, 10);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)